logging/logger_registry.go

```diff
@@ -59,22 +59,20 @@ func (lr *Registry) Update(logConfig []LoggerPatternConfig, warnLogger Logger) e
 
 	appliedConfigs := make(map[string]Level)
 	for _, lpc := range logConfig {
-		if !validatePattern(lpc.Pattern) {
-			warnLogger.Warnw("failed to validate a pattern", "pattern", lpc.Pattern)
+		r, err := regexp.Compile(buildRegexFromPattern(lpc.Pattern))
+		if err != nil {
+			warnLogger.Warnw("Log regex did not compile", "pattern", lpc.Pattern)
```

Comment:
Maybe include the error message from `Compile`.

Reply:
Good idea